driver.sh

```diff
@@ -87,6 +87,12 @@ setup_variables() {
       qemu_cmdline=( -cpu cortex-a57
                      -drive "file=images/arm64/rootfs.ext4,format=raw"
```

Comment:
I'm surprised this isn't already using arguments for virtio-blk (as in arm32_v7 above). It'd need:

                     -device "virtio-blk-device,drive=rootfs"

and adding "id=rootfs" to the -drive line above.

I'd note that I boot from /dev/vda1 not /dev/vda. A real partition might be needed for UEFI sanity? I'm not 100% sure if that's the cause of the failures.